### DISCUSSION TITLE
Leafmap.Map() is not showing up in the browser using Map.to_streamlit()

### DISCUSSION THREAD

**Discussion by PavlosDem99** [opengeos/leafmap#337] [General]
![image](https://user-images.githubusercontent.com/95135614/210827001-63961b11-2f3f-425b-bfaf-a33a184ed1c6.png)

I get this blank Map when I am trying to use Leafmap.Map() (foliumap.Map() as well), with Streamlit. 
My code: 

![image](https://user-images.githubusercontent.com/95135614/210827476-702614e8-8f65-4081-b2ec-0e86d63d00e1.png)

![image](https://user-images.githubusercontent.com/95135614/210827423-48c2be0d-a5e3-440b-8fe7-d1c5e63caea3.png)

Can anyone please help me with this?

**Comment by giswqs** [comment]
Are you using geemap or leafmap? Your code is a bit confusing. 

Try creating a simple leafmap without anything else

```python
import leafmap.foliumap as leafmap
import streamlit as st

m = leafmap.Map()
m.to_streamlit()
```

**Reply by PavlosDem99** [reply]
Thanks for your so fast response dr Wu. I installed both of them because leafmap didn't show up. 
I tried to make a leafmap Map without anything else and it works perfectly !

P.S. I wish you a happy new year

**Comment by PavlosDem99** [comment]
I found out that may a conflict appears with leafmap.kepler, when i import lefmap.kepler the same problem shows up.
![image](https://user-images.githubusercontent.com/95135614/210840524-8b7fcf2f-a980-4371-b6db-9bdc9d2775cc.png)

**Reply by giswqs** [reply]
kepler does not work with streamlit. Don't use it.
